nilearn/plotting/tests/test_img_plotting/test_plot_glass_brain.py

```diff
@@ -118,4 +126,12 @@ def test_plot_glass_brain_with_completely_masked_img(display_mode):
 
 def test_plot_glass_brain_vmin_vmax(img_3d_mni):
     """Smoke tests for plot_glass_brain being passed vmin and vmax."""
-    plot_glass_brain(img_3d_mni, vmin=-2, vmax=2)
+    # set plot_abs=False because otherwise negative vmin is not allowed
+    plot_glass_brain(img_3d_mni, vmin=-2, vmax=2, plot_abs=False)
```

Comment:
I think with the recent update vmin is allowed so you can revert the change in this test